.github/workflows/build.yml

```diff
@@ -0,0 +1,109 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ubuntu-gcc:
+    name: Ubuntu 18.04 / gcc
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+    - name: Install build dependencies
+      run: |
+        sudo apt-get update && sudo apt-get install -y --no-install-recommends \
+          ccache \
+          libavformat-dev \
+          libchromaprint-dev \
+          libebur128-dev \
+          libfaad-dev \
+          libfftw3-dev \
+          libflac-dev \
+          libid3tag0-dev \
+          liblilv-dev \
+          libmad0-dev \
+          libmodplug-dev \
+          libmp3lame-dev \
+          libmp4v2-dev \
+          libopus-dev \
+          libopusfile-dev \
+          libportmidi-dev \
+          libprotobuf-dev \
+          libqt5opengl5-dev \
+          libqt5sql5-sqlite \
+          libqt5svg5-dev \
+          libqt5x11extras5-dev \
+          librubberband-dev \
+          libshout3-dev \
+          libsndfile1-dev \
+          libsoundtouch-dev \
+          libsqlite3-dev \
+          libtag1-dev \
+          libupower-glib-dev \
+          libusb-1.0-0-dev \
+          libwavpack-dev \
+          portaudio19-dev \
+          protobuf-compiler \
+          qt5-default \
+          qt5keychain-dev \
+          qtscript5-dev \
+    - name: Set up cmake
+      uses: jwlawson/actions-setup-cmake@v1.4
+      with:
+        cmake-version: '3.13.x'
+    - name: Create build directory
+      run: mkdir cmake_build
+    - name: Set up ccache cache
+      uses: actions/cache@v2
+      env:
+        cache-name: ccache
+      with:
+        path: .cache/ccache
+        key: ${{ env.cache-name }}-${{ runner.os }}-${{ github.head_ref }}-${{ github.run_number }}
+        restore-keys: |
+          ${{ env.cache-name }}-${{ runner.os }}-${{ github.head_ref }}
```

Comment:
The setup of this action is customized for ccache. The indirection `${{ env.cache-name }}` does not help here. Why not use `ccache` literally here?

According to the [examples](https://github.com/actions/cache/blob/main/examples.md) the preferred or recommended pattern seems to be `${{ runner.os }}-ccache-*`.

Reply:
Fixed.